cprover_bindings/src/irep/to_irep.rs

```diff
@@ -162,6 +162,10 @@ impl ToIrep for Expr {
         } else {
             self.value().to_irep(mm).with_location(self.location(), mm).with_type(self.typ(), mm)
         }
+        .with_named_sub_option(
+            IrepId::CCSizeofType,
+            self.size_of_annotation().map(|ty| ty.to_irep(mm)),
+        )
```

Comment:
My internal type-checker is complaining: I'd expect a `size_of_annotation` to return an unsigned number (i.e., a `usize`), but it returns a type instead. The irep ID is `CCSizeofType`, but you're encoding an `Option<&Type>`?

Reply:
The actual size is the value of the expression, the annotation just tells you the size of which type was computed to arrive at that value.